FuSa-addon/scripts/REQIF_export.groovy

```diff
@@ -51,33 +44,46 @@ def traverseTree(thisNode, parent_identifier)
     }
 
     thisNode.children.each{
-        traverseTree(it, identifier)
+        traverseTree(it, identifier, requirementlists, relationlist)
     }
 }
 
-def makeHeader(XML, DATE)
+def makeHeader(XML)
 {
     XML.'THE-HEADER'{
-        XML.'REQ-IF-HEADER'('IDENTIFIER':'_adkamo'){
-            XML.'CREATION-TIME'(DATE)
-            XML.'REPOSITORY-ID'('?')
-            XML.'REQ-IF-TOOL-ID'('Freemind FuSa Extension v0.5')
-            XML.'REQ-IF-VERSION'('1.0')
-            XML.'SOURCE-TOOL-ID'('I guess this is only relevant for a round trip')
+        XML.'REQ-IF-HEADER'('IDENTIFIER':"_" + UUID.randomUUID().toString()){
+            XML.'CREATION-TIME'("${format.format(new Date())}")
+            XML.'REPOSITORY-ID'("__73f2bb4d-760e-44d9-99d3-5807d44b4928")
+            XML.'REQ-IF-TOOL-ID'("_5021db30-dd57-4612-bb75-98c4801619c0")
+            XML.'REQ-IF-VERSION'('1.1')
```

Comment:
According to the spec (v.1.2) this field always needs to contain "1.0"
That might be an error though, considering it is Spec version 1.2 ...
https://www.omg.org/spec/ReqIF/1.2/PDF (page 27)